glib/src/variant_type.rs

```diff
@@ -137,8 +137,7 @@ impl Deref for VariantType {
     type Target = VariantTy;
     fn deref(&self) -> &VariantTy {
         unsafe {
-            assert!(self.len > 0);
```

Comment:
This is already guaranteed by construction, just like that the stored pointer is always not `NULL`